Notepad/HomeViewController.swift

```diff
@@ -17,20 +17,32 @@ class HomeViewController: UIViewController {
     }
     
     @IBAction func handleSaveButtonPressed(_ sender: Any) {
-        let title = titleTextField.text
-        print("my title is: \(title!)")
         
-        let notes = bodyTextView.text
-        print("my body is: \(notes!)")
         
-        UserDefaults.standard.set(title, forKey: "Title")
-        UserDefaults.standard.set(notes, forKey: "Body")
+        let alert = UIAlertController(title: "Welcome", message: "Would you like to save your note?", preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "No", style: .cancel, handler: nil))
+    
+        alert.addAction(UIAlertAction(title: "Yes", style: .default, handler: { (UIAlertAction) in
+            if let title = self.titleTextField.text, let notes = self.bodyTextView.text {
```

Comment:
I would keep the naming nomenclature consistent - so `title`, and `body`.

Reply:
Anyways, is there a reason you are using `if let`, vs just setting the values to UserDefaults directly? For example:

```
UserDefaults.standard.set(self.titleTextField.text, forKey: "Title")
```

Reply:
tbh i just searched how to use uialertactions and followed the guide mostly

Reply:
You gotta know what your code is doing, instead of just following. You don't want to be a follower forever

Reply:
how else was i supposed to learn how to use it

Reply:
`if let` is fundamental Swift.
<img width="1477" alt="image" src="https://github.com/williamso1234/Notepad/assets/5104410/2447c33f-4bf5-44f5-881a-c0b18fb115d6">